opensrp-maternity/src/main/java/org/smartregister/maternity/interactor/BaseMaternityRegisterActivityInteractor.java

```diff
@@ -47,14 +47,14 @@ public BaseMaternityRegisterActivityInteractor() {
     }
 
     @Override
-    public void fetchSavedMaternityOutcomeForm(final @NonNull String baseEntityId, final @Nullable String entityTable, @NonNull final MaternityRegisterActivityContract.InteractorCallBack interactorCallBack) {
+    public void fetchSavedMaternityOutcomeForm(final @Nullable String formType, final @NonNull String baseEntityId, final @Nullable String entityTable, @NonNull final MaternityRegisterActivityContract.InteractorCallBack interactorCallBack) {
```

Comment:
Do you need this method?

Reply:
yes, because we try to first fetch the saved form and show it.